.gitlab-ci.yml

```diff
@@ -89,11 +89,16 @@ trigger:mender-dist-packages:
   stage: trigger
   before_script:
     - apk add --no-cache curl
+    - MENDER_CLIENT_VERSION=$(git tag | egrep -e '^[0-9]+\.[0-9]+\.[0-9]+$' | sort -V -r | head -n1)
+    # This will only be true in the case that we have a new newest client tag
+    - PUBLISH_LATEST_CLIENT_PACKAGE="false"
```

Comment:
shouldn't these be `export`ed?

Reply:
Hmm, why? They're not needed in any sub-processes, I think?

Reply:
They are not needed, as `before_script` and `script` are just concatenated and executed in the same shell.

If any, we could also remove `export` from line below

Reply:
I'll create a tidy-up PR

Reply:
https://github.com/mendersoftware/mender/pull/661

Reply:
the main reason I'm asking is https://gitlab.com/Northern.tech/Mender/mender/-/jobs/840156620 - trying it locally the comparison failed unless I went the `export` way...

Reply:
Hmm, really :thinking:  How can that be?

Reply:
@oleorhagen  After some investigation, the problem seems to be in the subshell, and my current hypothesis is that `git tag` returns an empty list because the GitLab checkout is a shallow clone. I asked @mzedel to follow-up :+1:

Reply:
great... now after debugging the git behavior in #662 and the `git tag`s being present already, I retried the failed job and => https://gitlab.com/Northern.tech/Mender/mender/-/jobs/840632341 succeeded... without changes...